operator/scripts/create_release_charts.sh

```diff
@@ -46,4 +46,5 @@ mkdir -p "${OUTPUT_DIR}"
 
 cp -R "${INSTALLER_DIR}/charts" "${OUTPUT_DIR}"
 cp -R "${INSTALLER_DIR}/profiles" "${OUTPUT_DIR}"
+cp -R "${INSTALLER_DIR}/examples" "${OUTPUT_DIR}"
```

Comment:
This will add examples to assets.gen.go which will end up in istioctl binary. Is this desirable?

We can fix this in release-builder repo.

Reply:
> This will add examples to assets.gen.go which will end up in istioctl binary. Is this desirable?

No, this should add `examples` folder in the downloaded release dir.

I'm unfamiliar with the `release-builder` code. I refer this line https://github.com/istio/release-builder/blob/649b2a51ed96da57084e2bb55865a1a17e1d86e7/pkg/build/archive.go#L64 

@jwendell Can you please point me to the right file?

Reply:
What I'm saying is that this script is called by `make gen-charts` [here](https://github.com/istio/istio/blob/master/Makefile.core.mk#L332) and [here](https://github.com/istio/istio/blob/master/operator/scripts/create_assets_gen.sh#L29). Therefore whatever is copied to `OUTPUT_DIR` will be present in the istioctl binary.

I've created a [PR](https://github.com/istio/release-builder/pull/364) in release-builder repo to fix this.

Reply:
Great, thanks @jwendell . I will close this PR.